src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -2018,6 +2018,10 @@ public ColumnFamily getColumnFamily(QueryFilter filter)
 
         ColumnFamily result = null;
 
+        if (filter.filter instanceof SliceQueryFilter) {
+            ((SliceQueryFilter) filter.filter).setMetrics(metric);
+        }
+
```

Comment:
this method of passing through the metrics is unfortunate; however, it is how metrics are currently recorded after-the-fact https://github.com/palantir/cassandra/pull/524/files#diff-9db147d2e310e83ba17e4b4c756ce0aa90ce1742006fa3bc2418aaaf1fe4b7c9L2070-L2072